app/admin/cdr/cdr_exports.rb

```diff
@@ -34,6 +36,12 @@
     render body: nil
   end
 
+  member_action :delete, method: :delete do
+    Worker::RemoveCdrExportFileJob.perform_later(resource.id)
+    flash[:notice] = 'The file will be deleted in async way!'
```

Comment:
@gigorok in this situation multiple delete possible, deletion should 
1) change status
2) create dj to delete file